src/main/java/codechicken/nei/LayoutManager.java

```diff
@@ -359,11 +359,18 @@ public void init() {
                 this.icons[2] = new DrawableBuilder("nei:textures/nei_tabbed_sprites.png", 64, 0, 16, 16).build();
             }
 
+            private boolean getIsAccessibleControlEventKey() {
+                if (Minecraft.isRunningOnMac) {
+                    return Keyboard.getEventKey() == Keyboard.KEY_LMETA || Keyboard.getEventKey() == Keyboard.KEY_RMENU;
```

Comment:
Is it intended that key is `RMENU` or one should be `RMETA`?

Reply:
It should be RMETA, RMENU is Right Option ⌥ key actually. Added commit.